doc/builders/packages/steam.section.md

```diff
@@ -56,7 +56,7 @@ Use `programs.steam.enable = true;` if you want to add steam to systemPackages a
 
 ## steam-run {#sec-steam-run}
 
-The FHS-compatible chroot used for Steam can also be used to run other Linux games that expect a FHS environment. To use it, install the `steam-run-native` package and run the game with
+The FHS-compatible chroot used for Steam can also be used to run other Linux games that expect a FHS environment. To use it, install the `steam-run` package and run the game with
```

Comment:
Since the non-native is non-free, is there any alternative for programs expecting an FHS environment?

Reply:
+1! I don't understand why this was removed. it was very useful to me. programs like `minigalaxy` shouldn't require unfree dependencies

Reply:
We probably shouldn't be using a steam runtime for `minigalaxy`, steam itself is unfree, so it's expected to need unfree to use steam (or downstream package)